### PR TITLE
fix: switch FBSDK logTime to number

### PR DESCRIPTION
**Pull request by oscb**
- Changes `_logTime` in FBSDK plugin to use Unix Timestamp (in number format, non-decimal).
- Changes default to `undefined` (edge case)
- Fixes an issue in the previous implementation: `_logTime` would never be set and always relied on the default value instead of using the `event.timestamp`.
- Fixed and added tests.

### CODE REVIEW

**Review by alanjcharles — APPROVED** [review]

File: packages/plugins/plugin-facebook-app-events/src/FacebookAppEventsPlugin.ts [hunk 1]
```diff
@@ -147,6 +148,7 @@ export class FacebookAppEventsPlugin extends DestinationPlugin {
     const safeEvent = mappedPropNames(
       event as unknown as Record<string, unknown>
     );
+    console.log(safeEvent);
```
Comment (esdrasetrenne) [inline packages/plugins/plugin-facebook-app-events/src/FacebookAppEventsPlugin.ts:151]: log needed?

Comment (oscb) [reply packages/plugins/plugin-facebook-app-events/src/FacebookAppEventsPlugin.ts]: Thanks for spotting this!
